src/VisualStudio/CSharp/Impl/LanguageService/CSharpCodeCleanupFixer.cs

```diff
@@ -383,12 +386,23 @@ async Task<Solution> ApplyFixAsync(ProgressTracker progressTracker, Cancellation
             }
         }
 
-        private async Task<bool> FixAsync(Workspace workspace, Func<ProgressTracker, CancellationToken, Task<Solution>> applyFixAsync, ICodeCleanUpExecutionContext context, CancellationToken cancellationToken)
+        private async Task<bool> FixAsync(Workspace workspace, Func<ProgressTracker, CancellationToken, Task<Solution>> applyFixAsync, ICodeCleanUpExecutionContext context,
+            string contextName, CancellationToken cancellationToken)
         {
             using (var cancellationTokenSource = CancellationTokenSource.CreateLinkedTokenSource(context.OperationContext.UserCancellationToken, cancellationToken))
```

Comment:
I think we need to create this linked cancellation after new scope? otherwise, cancellation of the new scope might not get included in the linked cancellation token.

I need to take a look at OperationContext to be sure though. it might not matter if it returns same cancellation token (I think it does when using allowCancellationToken), but I think it would save some confusion if the linked cancellation is after AddScope for full solution load.